packages/vats/src/core/lib-boot.js

```diff
@@ -47,7 +47,7 @@ const setDiff = (a, b) => a.filter(x => !b.includes(x));
 /**
  * @param {import('@agoric/swingset-vat').VatPowers & {
  *   D: DProxy;
- *   logger: (msg) => void;
+ *   logger?: typeof console.log;
```

Comment:
Oh this one is a can of worms from what I recall. Some tools/tests pass different things in here. Types only so it may be fine, but there's no guarantee it might not blow up in the future.